examples/front_example.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
```

Comment:
Add the example folder where I plan to group all the examples of how to use it. In this case, I simply project the cameras.